bitnami/elasticsearch/templates/master/statefulset.yaml

```
@@ -83,9 +83,9 @@ spec:
       {{- if .Values.master.terminationGracePeriodSeconds }}
       terminationGracePeriodSeconds: {{ .Values.master.terminationGracePeriodSeconds }}
       {{- end }}
-      {{- if or (or .Values.initContainers .Values.master.initContainers) .Values.security.enabled .Values.sysctlImage.enabled (and .Values.volumePermissions.enabled .Values.master.persistence.enabled) }}
+      {{- if and (default true .Values.enableDefaultInitContainers) (or (or .Values.initContainers .Values.master.initContainers) (and (default true .Values.enableDefaultInitContainers) (or .Values.security.enabled .Values.sysctlImage.enabled (and .Values.volumePermissions.enabled .Values.master.persistence.enabled)))) }}
```

If I am not wrong this condition will work in the same way and it's simpler
```suggestion
      {{- if and .Values.enableDefaultInitContainers (or (or .Values.initContainers .Values.master.initContainers) .Values.security.enabled .Values.sysctlImage.enabled (and .Values.volumePermissions.enabled .Values.master.persistence.enabled)) }}
```